app/Foliage/CmdBuild.hs

```diff
@@ -133,6 +135,7 @@ buildAction
             (IndexPkgMetadata pkgId)
             (fromMaybe currentTime pkgTimestamp)
 
```

Comment:
Then we can use the `Map PackageName [(Timestamp, VersionRange)]` created above to create the actual tar entries here. Perhaps let's call them `extraEntries` following hackage and cabal terminology.